app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -3,20 +3,22 @@ child :content_facet => :content_facet_attributes do
 
   node do |content_facet|
     content_facet.single_content_view # just so Rails gives a warning and we remember to fix this
-    version = content_facet.content_view_environments.first.content_view_version
-    {
-      :content_view_version => version.version,
-      :content_view_version_id => version.id,
-      :content_view_version_latest => version.latest?
-    }
+    version = content_facet.content_view_environments.first&.content_view_version
+    if version.present?
+      {
+        :content_view_version => version.version,
+        :content_view_version_id => version.id,
+        :content_view_version_latest => version.latest?
+      }
+    end
   end
 
   node :content_view_default? do |content_facet|
-    content_facet.single_content_view.default?
+    !!content_facet.single_content_view&.default?
```

Comment:
This seems a very javascripty thing and not ruby?
Why is this needed. The code prior to your single-cv change
`content_facet.content_view.default?` seems to think content facet has to have a CV

Reply:
`content_view_default?` should return a Boolean; I wanted to ensure it's not `nil`. Switched to `::Foreman::Cast.to_bool`.

Reply:
...and now  to `|| false`